### DISCUSSION TITLE
DNS issue

### DISCUSSION THREAD

**Discussion by badblli** [vercel/community#3639] [Help]
### Summary

@root:~# dig A depoantalya.com +short
76.76.21.61
76.76.21.142

vercel is resolving two different ip addresses on my website. For this reason, I sometimes get the error that my website cannot be reached, I guess because the threads are conflicting. wouldn't it normally have to resolve a single ip address, namely 76.76.21.21?

how can i fix this?

### Example

https://www.whatsmydns.net/#A/depoantalya.com

### Steps to Reproduce

@root:~# dig A depoantalya.com +short
76.76.21.61
76.76.21.142

**Comment by amyegan** [comment]
Hey @badblli. When I run `dig A depoantalya.com +short`, I get different IP addresses from what you see (`34.73.83.172` and `34.148.19.16`) and the https://www.whatsmydns.net/#A/depoantalya.com page shows various different IP addresses across different DNS servers. It looks like there are multiple different registrars' nameservers associated with the domain https://www.whatsmydns.net/#NS/depoantalya.com

Were the records for the domain very recently changed? It is normal for DNS settings to take up to 48 hours to fully propagate through the global network of DNS servers.

**Comment by badblli** [comment]
I switched to netlify. problem fixed. This was an ongoing problem for 3 days.


but i still love vercel <3
